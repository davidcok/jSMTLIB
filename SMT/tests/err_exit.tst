; exit with unexpected arguments
(exit x)
