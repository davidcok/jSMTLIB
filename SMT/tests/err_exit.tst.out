(exit x)
      ^
(error "A exit command takes no arguments")
